src/tests/unit/tests/injected/visualization/drawer.test.ts

```diff
@@ -1153,7 +1153,7 @@ describe('Drawer', () => {
                 throw new Error('Not implemented');
             }
 
-            public getDialogRenderer() {
+            public getDialogRenderer(): any {
```

Comment:
> any [](start = 39, length = 4)

Can you instead update `Formatter` to use `DialogRenderer?` and match that type here?

Reply:
Done.